### PR TITLE
ci: Add a PR comment w/ the published alpha versions

### PR DESCRIPTION
**Pull request by jakemhiller**
This is a little heavy-handed, I couldn't really find something that did what I wanted but I didn't have time to write something. Might come back and write a small open-source tool to handle this commenting stuff.

In the meantime this works :)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -153,6 +153,27 @@ jobs:
...
+              docker run --rm \
+                      -e GITHUB_TOKEN \
+                      -e GITHUB_OWNER="${CIRCLE_PROJECT_USERNAME}" \
+                      -e GITHUB_REPO="${CIRCLE_PROJECT_REPONAME}" \
+                      -e GITHUB_COMMENT_TYPE=pr \
+                      -e GITHUB_PR_ISSUE_NUMBER="${CIRCLE_PR_NUMBER:-${CIRCLE_PULL_REQUEST##*/}}" \
+                      -e GITHUB_COMMENT_TEMPLATE='<h3>Published Alpha Packages:</h3><pre>{{$x := splitList "\n" .}}{{range $x}}{{$y := split ":" .}}{{printf "%s@%v<br/>" $y._1 $y._2}}{{end}}</pre><!--LERNA_VERSION_COMMENT-->' \
+                      -e GITHUB_DELETE_COMMENT_REGEX="LERNA_VERSION_COMMENT" \
+                      -e GITHUB_COMMENT="${ALPHA_PACKAGES}" \
+                      cloudposse/github-commenter:0.5.0-58
```
Comment (JoshuaKGoldberg) [inline .circleci/config.yml]: > cloudposse

I love it

File: .circleci/config.yml [hunk 2]
```diff
@@ -153,6 +153,27 @@ jobs:
...
+            then
+              echo "No alpha published packages found, will not add PR comment"
+            else
+              docker run --rm \
+                      -e GITHUB_TOKEN \
+                      -e GITHUB_OWNER="${CIRCLE_PROJECT_USERNAME}" \
+                      -e GITHUB_REPO="${CIRCLE_PROJECT_REPONAME}" \
+                      -e GITHUB_COMMENT_TYPE=pr \
+                      -e GITHUB_PR_ISSUE_NUMBER="${CIRCLE_PR_NUMBER:-${CIRCLE_PULL_REQUEST##*/}}" \
+                      -e GITHUB_COMMENT_TEMPLATE='<h3>Published Alpha Packages:</h3><pre>{{$x := splitList "\n" .}}{{range $x}}{{$y := split ":" .}}{{printf "%s@%v<br/>" $y._1 $y._2}}{{end}}</pre><!--LERNA_VERSION_COMMENT-->' \
```
Comment (JoshuaKGoldberg) [inline .circleci/config.yml]: Could you add an emoji? 😄

**Comment by codecademydev** [comment]
<h3>💌Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.1.5<br/>@codecademy/gamut-styles@2.2.11<br/>@codecademy/gamut-templates@2.2.30<br/>@codecademy/gamut@2.4.6<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.1.5<br/>@codecademy/gamut-styles@2.2.11<br/>@codecademy/gamut-templates@2.2.30<br/>@codecademy/gamut@2.4.6<br/></pre>

**Pull request merged**